apps/builder/app/canvas/features/build-mode/block.tsx

```diff
@@ -81,7 +83,9 @@ export const Block = React.forwardRef<
   return (
     <div ref={ref} style={editableBlockStyle} {...props}>
```

Comment:
why do you want to render the div still?

Reply:
data-ws-id, selectors, ref's are used in various algorithms we have. Every our control has some kind of root element with all that stuff